cea/constants.py

```diff
@@ -18,7 +18,7 @@
 
 
 #Shapefiles precision of decimals
-SHAPEFILE_TOLERANCE = 6 #this is precision of milimiters
+SHAPEFILE_TOLERANCE = 20 #this is precision of milimiters
```

Comment:
this value is not consistent with the description in the PR ('15' was indicated there)
also, is it necessary to change the default from 6 to 15? 
is it a common case that tolerance is exceeded?
if yes, please update the documentation to describe what does '15' means (e.g., 6 is the precision of millimeters)

Reply:
@Shanshanhsieh  I have tested multiple case studies and with 20 mm of SHAPEFILE_TOLERANCE they were no errors, while with 6mm of SHAPEFILE_TOLERANCE there were. I have talked to Jimeno about this and he said there is no harm in increasing the SHAPEFILE_TOLERANCE by this much. Rather a too small Tolerance would be problematic since for more case studies this would throw and error.